src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-tabs/GeneViewTabs.scss

```diff
@@ -0,0 +1,42 @@
+@import 'src/styles/common';
+
+.geneTab {
+  background-color: #33adff;
+  color: #fff;
+  font-weight: 700;
+  border-radius: 3px;
+  margin-right: 3px;
+  height: 30px;
+  &::after {
+    border: none;
+  }
+}
+
+.disabledGeneTab {
+  background-color: $grey;
+}
+
+.selectedGeneTabName {
+  background-color: transparent;
+  border: 1px solid $blue;
+
+  &::after {
+    width: 12px;
+    height: 12px;
+    background-color: $black;
+    box-shadow: none;
+    box-sizing: border-box;
+    content: '';
+    position: absolute;
+    left: calc(50% - 8px);
+    transform-origin: 0 0;
+    transform: rotate(-45deg);
+    top: 28px;
+  }
+}
```

Comment:
Apparently, the arrow drops a shadow in the mockup 😨 

![image](https://user-images.githubusercontent.com/6834224/76531070-8fcc5500-646c-11ea-812c-ee5a415d30b2.png)

Reply:
Added the shadow 👍